app/views/transaction.py

```diff
@@ -0,0 +1,766 @@
+            if not fund_transfer_data:
+                raise FundTransferObjectNotFound("Fund transfer with this id does not exist")
+
+            fund_transfer_data.from_account = data.get('from_account', fund_transfer_data.from_account)
+            fund_transfer_data.to_account = data.get('to_account', fund_transfer_data.to_account)
+
+            db.session.commit()
+            result = fund_transfer_schema.dump(fund_transfer_data)
+
+            logger.info("Response for put with id request for fund transfer {}".format(result))
+            response = ResponseGenerator(data=result,
+                                         message="Fund transfer with this id updated successfully",
+                                         success=True,
+                                         status=HTTPStatus.OK)
+            return response.success_response()
+        except FundTransferObjectNotFound as err:
+            logger.exception(err.message)
+            response = ResponseGenerator(data={},
+                                         message=err.message,
+                                         success=False,
+                                         status=HTTPStatus.NOT_FOUND)
+            return response.error_response()
+
+    def delete(self, fund_transfer_id):
+        """
+             This is DELETE API
+             Call this api passing a fund transfer id
+             parameters:
+                 source: String
+                 destination: String
+             responses:
+                 404:
+                     description: Fund transfer with this id does not exist
+                 200:
+                     description: Fund transfer with this id deleted successfully
+                     schema:
+                         FundTransferSchema
+         """
+        try:
+            fund_transfer_data = FundTransfer.query.filter(FundTransfer.id == fund_transfer_id).first()
+            if not fund_transfer_data:
+                raise FundTransferObjectNotFound("Fund transfer with this id does not exist")
+
+            db.session.delete(fund_transfer_data)
+            db.session.commit()
+
+            logger.info("Response for delete request for fund transfer:"
+                        "Fund transfer with this id deleted successfully")
+
+            return "Fund transfer record deleted successfully"
+        except FundTransferObjectNotFound as err:
+            logger.exception(err.message)
+            response = ResponseGenerator(data={},
+                                         message=err.message,
+                                         success=False,
+                                         status=HTTPStatus.NOT_FOUND)
+            return response.error_response()
+
+
+class MiniStatementResources(Resource):
+    def get(self):
+        try:
+            mini_statement_data = AccountTransactionDetails.query.order_by(desc(AccountTransactionDetails.id)).limit(10)
```

Comment:
Mini statement is usually given for given account so we should accept account number/id and give transaction for only that account.

Reply:
Made the changes as suggested.